.travis.yml

```diff
@@ -25,7 +25,18 @@ matrix:
           packages:
             - libaio1
             - rpm
+      sudo: required
+    - env:
```

Comment:
here is how I would like set up the env:
https://github.com/jayvdb/travis-oracle/commit/b46d9d94af7d1a406254297e6320bd672c36211c